workflows/argo/training.yaml

```diff
@@ -23,7 +23,7 @@ spec:
           - {name: memory, value: 6Gi}
           - {name: flags, value: " "}
       container:
-        image: us.gcr.io/vcm-ml/fv3fit
+        image: us.gcr.io/vcm-ml/prognostic_run
```

Comment:
Given the tests for this are run and docs are built in the fv3net image/environment, would it make more sense to run the training argo workflow in that image?